components/tools/OmeroJava/test/integration/RenderingEngineTest.java

```diff
@@ -1910,6 +1910,7 @@ public void testRenderingEngineProjectionDef() throws Exception {
         pDef.setActive(omero.rtypes.rbool(true));
         pDef.setStartPlane(omero.rtypes.rint(0));
         pDef.setEndPlane(omero.rtypes.rint(1));
+        pDef.setStepping(omero.rtypes.rint(1));
```

Comment:
maybe pick a different number for each property? then can rule out property values somehow getting swapped.